routes/topics.js

```diff
@@ -16,6 +16,7 @@ router.get('/follow', isAuth, topicsController.getFollowTopic);
 router.get('/latest', validate(TopicValidation.latestPost), isAuth, topicsController.getLatestPost);
 router.put('/follow', isAuth, topicsController.putFollowTopic);
 router.put('/follow-v2', isAuth, topicsController.followTopicV2);
+router.post('/check-name', validate(TopicValidation.checkName), isAuth, topicsController.checkName);
```

Comment:
@brm-stnd @ibramastana 
1. use more explicit name for route (for example `/is-exist`)
2. if only for checking, use `router.get`

Reply:
fixed